code/modules/spacepods/spacepod.dm

```diff
@@ -178,10 +177,20 @@
 			to_chat(M, S)
 	if(!health)
 		spawn(0)
+			if(armor_stage == 0)
+				var/datum/effect/system/harmless_smoke_spread/smoke = new
+				smoke.set_up(1,0, src, 0)
+				smoke.start()
 			if(passengers)
 				for(var/mob/M in passengers)
 					to_chat(M, "<span class='userdanger'>Critical damage to the vessel detected, core explosion imminent!</span>")
 			for(var/i = 10, i >= 0; --i)
+				if(armor_stage == 0)
+					var/datum/effect/system/harmless_smoke_spread/smoke = new
+					smoke.set_up(1,0, src, 0)
+					smoke.start()
+					for(var/mob/M in passengers)
+						to_chat(M, "<span class='boldnotice'>Gas vented. Explosion averted.</span>")
```

Comment:
also wtf is this even
armor_stage is 0 when the armor is fully on right
so why does that max it completely immortal